Volume/Views/MainView/PublicationList.swift

```diff
@@ -76,8 +69,8 @@ struct PublicationList: View {
                             }
                         }
                         .padding([.leading, .trailing], 10)
-                    case .results(let results):
-                        LazyHStack(spacing: 12) {
+                    case .reloading(let results), .results(let results):
+                        HStack(spacing: 12) {
```

Comment:
So we want view to display previous results while it is reloading?  Just want to clarify because I thought we were implementing different behavior but this works too

Reply:
Yeah, it would be weird to remove the currently loaded results when making a network request imo